### PR TITLE
Updated cmake lists to use with pls build tool

### PR DESCRIPTION
**Pull request by amdrozdov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,12 @@ build/
...
 googletest/
 wsServer/
+
+# Added automatically by `pls`. Okay to push to git.
+.debug
+.pls
+.release
+.vscode
+
+# Added automatically by `pls`. Okay to push to git.
```
Comment (dkorolev) [inline .gitignore]: Let's clean this up?

Comment (dkorolev) [reply .gitignore]: I.e. merge the sections manually?

**Review by dkorolev — APPROVED** [review]

**Comment by dkorolev** [comment]
LGTM++

I'd say remove that comment line and re-sort `.gitignore`-s, and merge right in!

**Comment by amdrozdov** [comment]
Done

**Pull request merged**